app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -131,41 +132,52 @@ private void updateCallToAction() {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(pair -> {
-                            WikipediaApp app = WikipediaApp.getInstance();
-                            PageTitle captionSourcePageTitle = new PageTitle(imageTitle, new WikiSite(Service.COMMONS_URL, getTitle().getWikiSite().languageCode()));
-                            ImageInfo imageInfo = pair.second.query().firstPage().imageInfo();
-
-                            if (!pair.first.containsKey(getTitle().getWikiSite().languageCode())) {
-                                pageHeaderView.setUpCallToAction(app.getResources().getString(R.string.suggested_edits_article_cta_image_caption));
-                                callToActionSourceSummary = new SuggestedEditsSummary(captionSourcePageTitle.getPrefixedText(), getTitle().getWikiSite().languageCode(), captionSourcePageTitle,
-                                        captionSourcePageTitle.getDisplayText(), StringUtils.defaultIfBlank(StringUtil.fromHtml(imageInfo.getMetadata().imageDescription()).toString(), null),
-                                        imageInfo.getThumbUrl());
-
-                                return;
-                            }
-                            if (app.language().getAppLanguageCodes().size() >= MIN_LANGUAGES_TO_UNLOCK_TRANSLATION) {
-                                for (String lang : app.language().getAppLanguageCodes()) {
-                                    if (!pair.first.containsKey(lang)) {
-                                        callToActionIsTranslation = true;
-                                        PageTitle captionTargetPageTitle = new PageTitle(imageTitle, new WikiSite(Service.COMMONS_URL, lang));
-                                        String currentCaption = pair.first.get(getTitle().getWikiSite().languageCode());
-                                        captionSourcePageTitle.setDescription(currentCaption);
-                                        callToActionSourceSummary = new SuggestedEditsSummary(captionSourcePageTitle.getPrefixedText(), captionSourcePageTitle.getWikiSite().languageCode(), captionSourcePageTitle,
-                                                captionSourcePageTitle.getDisplayText(), currentCaption, getLeadImageUrl());
-
-                                        callToActionTargetSummary = new SuggestedEditsSummary(captionTargetPageTitle.getPrefixedText(), captionTargetPageTitle.getWikiSite().languageCode(), captionTargetPageTitle,
-                                                captionTargetPageTitle.getDisplayText(), null, getLeadImageUrl());
-                                        pageHeaderView.setUpCallToAction(app.getResources().getString(R.string.suggested_edits_article_cta_image_caption_in_language, app.language().getAppLanguageLocalizedName(lang)));
-                                        break;
-                                    }
+                    if (pair != null) {
```

Comment:
The `pair` parameter will never be null, so this check is not necessary.

Reply:
Done!